### ISSUE TITLE
Fix "SecurityError: SecurityError: DOM Exception 18" error

### ISSUE THREAD

**Issue by mgoria** [MitocGroup/deep-framework#174]
```
User-Agent	
Mozilla/5.0 (iPhone; CPU iPhone OS 9_2 like Mac OS X) AppleWebKit/601.1.46 (KHTML, like Gecko) Version/9.0 Mobile/13C75 Safari/601.1
```

```javascript 
null: SecurityError: SecurityError: DOM Exception 18
  at ? (/js/lib/deep-framework.js:25:17458)
```

**Comment by mgoria** [comment]
This issue appears in Safari if `Cookies and website data` are set as `Always block` (see below screenshot). Looking for a graceful degradation solution ...

<img width="1274" alt="screen shot 2016-01-05 at 4 25 48 pm" src="https://cloud.githubusercontent.com/assets/2959952/12117362/05e0f24a-b3c9-11e5-9a62-61bef7dbd55d.png">

**Comment by mgoria** [comment]
This issue is fixed in `deep-framework@1.5.13`
